hash/msx2_flop.xml

```diff
@@ -17919,6 +18247,19 @@ HOMEBREW
 		</part>
 	</software>
 
+	<software name="kiboushi">
+		<description>Kibou Shima no Satsujin (Japan)</description>
+		<year>1995</year>
+		<publisher>Chirorin House</publisher>
+		<info name="alt_title" value="希望島の殺人"/>
```

Comment:
I’m not sure on the reading for the title here – is there any media showing furigana?